app/controllers/suscribir/suscripciones_controller.rb

```diff
@@ -35,8 +35,13 @@ def desuscribir
 
       suscripcion.destroy
       SuscripcionMailer.desuscribir(suscripcion, url_resuscripcion).deliver
-      redirect_to baja_realizada_path(type: clase, suscribible_id: suscripcion.suscribible_id,
-                                      email: encoded_email, token: token_bueno)
+      respond_to do |format|
+        format.html do
+          redirect_to baja_realizada_path(type: clase, suscribible_id: suscripcion.suscribible_id,
+                                          email: encoded_email, token: token_bueno)
+        end
+        format.js { head :no_content }
```

Comment:
The magic happens there -> https://github.com/Soluciones/Rankia/blob/show-suscripciones-perfil-usuario/app/assets/javascripts/suscripciones_ajax.js.coffee#L3